Lib/json/tool.py

```diff
@@ -1,22 +1,10 @@
-r"""Command-line tool to validate and pretty-print JSON
-
-Usage::
-
-    $ echo '{"json":"obj"}' | python -m json.tool
-    {
-        "json": "obj"
-    }
-    $ echo '{ 1.2:3.4}' | python -m json.tool
-    Expecting property name enclosed in double quotes: line 1 column 3 (char 2)
-
-"""
 import argparse
```

Comment:
No reason to drop the `json.tool` docstring:

```suggestion
"""Command-line tool to validate and pretty-print JSON
Usage::
    $ echo '{"json":"obj"}' | python -m json.tool
    {
        "json": "obj"
    }
    $ echo '{ 1.2:3.4}' | python -m json.tool
    Expecting property name enclosed in double quotes: line 1 column 3 (char 2)
"""
import argparse
```

Reply:
On seconds, thoughts, let's reword the implementation module docstring and reference the __main__.py docstring for the usage example.